src/util/address.rs

```diff
@@ -100,7 +98,7 @@ impl Address {
     /// This address type is discouraged as it uses more space but otherwise equivalent to p2pkh
     /// therefore only adds ambiguity
     #[inline]
-    pub fn p2upkh(pk: &PublicKey, network: Network) -> Address {
+    pub fn p2pkh_uncompressed(pk: &PublicKey, network: Network) -> Address {
```

Comment:
The name was fine as is.

Reply:
Yeah like I said, this sneaked in when rebasing off master. It's more consistent with the rest of the uncompressed variant methods, but I'm fine with changing it back.